src/app/children/child-details/child-details.component.ts

```diff
@@ -75,12 +75,12 @@ export class ChildDetailsComponent implements OnInit {
     this.form = this.fb.group({
       name:           [{value: this.child.name,           disabled: !this.editing}, Validators.required],
       // gender:         [{value: this.child.gender}], // reactive forms seem broken for mat-select, using ngModel instead
-      projectNumber:  [{value: this.child.projectNumber,  disabled: !this.editing}],
-      dateOfBirth:    [{value: this.child.dateOfBirth,    disabled: !this.editing}],
+      projectNumber:  [{value: this.child.projectNumber,  disabled: !this.editing}, Validators.required],
```

Comment:
The users may want to add candidates to the app, who are not enrolled in the project yet and don't have an official projectNumber. They could assign some other temporary numbers of course but it would be better to not enforce artificial constraints.

I think apart from name only the center makes sense to be required. Other information may not be available to the social workers yet when they first enter a child

Reply:
i removed the _required_ tags from all attributes except for name and center